store/blocks/home/home.jsonc

```diff
@@ -15,10 +15,6 @@
     ]
   },
 
-  "shelf#home": {
```

Comment:
Don't we need to declare the `list-context.product-list` instead? 🤔

Reply:
noo.. this block is only declared but it is not being used anywhere. The `list-context.product-list` is already declared properly